src/sidebars/version-directive.js

```diff
@@ -0,0 +1,12 @@
+(function() {
+    'use strict';
+
+    angular.module('openfin.version')
+        .value('version', '0.1.0')
```

Comment:
No real rationale, just a number sub-1 since we're not on public release yet (and it'll change shortly anyway). I'll look into grunt tasks for releasing.